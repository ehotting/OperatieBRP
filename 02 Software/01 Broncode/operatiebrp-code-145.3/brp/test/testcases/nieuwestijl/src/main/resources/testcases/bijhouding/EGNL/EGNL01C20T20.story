Meta:
@status                 Klaar
@regels                 R1860
@usecase                UCS-BY.HG

Narrative:
R1860 Datum einde H/GP moet geldige kalenderdatum zijn bij einde in NL

Scenario: Datum einde waarbij mm onbekend is (2005-00-01)
          LT: EGNL01C20T20

Given alle personen zijn verwijderd
Given enkel initiele vulling uit bestand /LO3PL-EGNL/EGNL01C70-April.xls

Given pas laatste relatie van soort 2 aan tussen persoon 594645001 en persoon 212737065 met relatie id 2000003 en betrokkenheid id 2000004

When voer een bijhouding uit EGNL01C20T20.xml namens partij 'Gemeente BRP 1'

Then is het foutief antwoordbericht gelijk aan /testcases/bijhouding/EGNL/expected/EGNL01C20T20.txt