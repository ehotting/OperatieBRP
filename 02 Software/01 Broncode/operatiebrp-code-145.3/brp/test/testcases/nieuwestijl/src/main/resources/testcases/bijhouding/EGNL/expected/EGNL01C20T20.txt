cvc-pattern-valid: Value '2005-00-01' is not facet-valid with respect to pattern '[0-9]{4}(-(0[1-9]{1}|1[0-2]{1})(-(0[1-9]{1}|[1-2]{1}[0-9]{1}|3[0-1]{1}))?)?' for type 'DatumMetOnzekerheid-s'.